elastic/src/main/scala/metrics.scala

```diff
@@ -17,13 +17,18 @@
 package funnel
 package elastic
 
+//Needed for static initialization of JVM Metrics
 import instruments._
+import scala.concurrent.duration._
 
 object metrics {
-  val NonHttpErrors    = counter("elastic/errors/other")
-  val HttpResponse5xx  = counter("elastic/http/5xx")
-  val HttpResponse4xx  = counter("elastic/http/4xx")
-  val HttpResponse2xx  = lapTimer("elastic/http/2xx")
-  val BufferDropped    = counter("elastic/queue/dropped")
-  val BufferUsed       = numericGauge("elastic/buffer/pending", 0)
+  //report these metrics every 30 seconds
+  val i = new Instruments(bufferTime = 30 seconds)
```

Comment:
The default report timing is 200 ms.  This was effectively producing a constant stream of POSTs to Elastic Search, as every 2xx would update the metric in sort of a perpetual motion machine.